portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/documentlibrarystore/GCSStore.testcase

```diff
@@ -81,7 +81,90 @@ definition {
 		AssertConsoleTextNotPresent(value1 = "java.io.IOException");
 	}
 
-	@ignore = "true"
+	@description = "Verify a data migration can be executed from the default store to Google Cloud store."
+	@priority = 3
+	test GCSStoreDataMigration {
+		property store.migration.test = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("Given a document is added to the default store") {
+			var bucketId = PropsUtil.get("gcs.bucket.id");
```

Comment:
@catarineaguiar you should move this out of the task since it is unused still. The best usage for readability is right before you use the declared variable, i would also be ok with seperating it into the beginnng as well. Its fine for this pull since we need to close this for the epic asap